src/software/ai/evaluation/possession_test.cpp

```diff
@@ -188,7 +188,8 @@ TEST(PossessionEvaluationTest, team_had_possession_half_second_ago)
     Robot robot2 = Robot(0, Point(-1, 3), Vector(), Angle::zero(),
                          AngularVelocity::zero(), Timestamp::fromSeconds(2));
 
-    world.updateBallState(BallState({0.1, 2.5}, {0, 10}, Timestamp::fromSeconds(3.5)));
+    world.updateBallState(
+        BallStateWithTimestamp({0.1, 2.5}, {0, 10}, Timestamp::fromSeconds(3.5)));
```

Comment:
Bit of a nit; but could we rename to `TimestampedBallState`? I think in general we follow this pattern more in the codebase (leading with the specialization).

Reply:
renamed